app/routes/sitemap.js

```diff
@@ -0,0 +1,63 @@
+//https://github.com/EnCiv/enciv-home/issues/23
+
+import { Iota } from 'civil-server'
+import fs from 'fs'
+import { SitemapStream } from 'sitemap'
+
+// Function to read JSON data from file
+function readJSONFile(filePath) {
+  try {
+    const data = fs.readFileSync(filePath, 'utf8')
+    console.log(`Successfully read data from ${filePath}`)
+    return JSON.parse(data)
+  } catch (err) {
+    console.error('Error reading JSON file:', err)
+    return null
+  }
+}
+
+// Export the getSitemap function as default
+export default async function getSitemap() {
+  try {
+    const iotas = await Iota.find({ path: { $exists: true } })
+    //console.log('Fetched Iotas from database:', iotas)
+
+    // Read data from iotas.json file
+    const data = readJSONFile('iotas.json')
```

Comment:
This is reading in iotas from Iota and also iotas.json.  Shouldn't read in iotas.json anymore - and in the common case both will have the same data.

Reply:
Working on this. I accidently used the iotas.json because of the same name. Now I changed it to iotas object.